Emby.Server.Implementations/LiveTv/TunerHosts/HdHomerun/HdHomerunManager.cs

```diff
@@ -1,4 +1,5 @@
 #pragma warning disable CS1591
+#pragma warning disable SA1649
```

Comment:
I think we should fix this, not disable it

Reply:
@Bond-009 I think you advised to add this #pragma, but why? Shouldn't we just keep the warnings until some point in the future when we decide to fix it?

Reply:
The idea was that we want to lower the amount of warnings now, but that in the Emby namespaces, they should only be fixed once the classes are moved to the Jellyfin namespace.